game/scripts/vscripts/items/black_king_bar.lua

```diff
@@ -18,6 +18,7 @@ function item_black_king_bar_1:OnSpellStart()
 	} )
 	caster:EmitSound( "DOTA_Item.BlackKingBar.Activate" )
 
+  caster:Purge( false, true, false, false, true )
```

Comment:
I'm pretty sure BKB's purge is a basic dispel, not a strong one. The last parameter is whether to purge things that require a strong dispel.